shortid_test.go

```diff
@@ -1,16 +1,15 @@
 // Copyright (c) 2016-2017. Oleg Sklyar & teris.io. All rights reserved.
 // See the LICENSE file in the project root for licensing information.
 
-package shortid_test
+package shortid
```

Comment:
Why is this change required? The test was only using the public API and that seems to be unchanged, so why merge the packages?